local/network.go

```diff
@@ -323,12 +332,15 @@ func (ln *localNetwork) addNode(nodeConfig node.Config) (node.Node, error) {
 		return nil, fmt.Errorf("repeated node name %s", nodeConfig.Name)
 	}
 
-	// [tmpDir] is where this node's config file, C-Chain config file,
+	if ln.rootDir == "" {
+		ln.log.Warn("no network root directory defined; will create this node's runtime directory in working directory")
+	}
```

Comment:
This shouldn't ever happen, right?

Reply:
Not if used in the normal flow, but just to make sure, as this is in `addNode`, and someone could be calling it via `AddNode` without the default config?